package.json

```diff
@@ -80,7 +80,7 @@
     "ember-route-action-helper": "^2.0.3",
     "ember-sinon": "^1.0.1",
     "ember-source": "^2.18.2",
-    "ember-svg-jar": "^1.1.1",
+    "ember-svg-jar": "ivanvotti/ember-svg-jar#custom-svgo-module",
```

Comment:
This branch has the custom SVGO module feature, that will be available in the next SVGJar release.